src/include/main/connection.h

```diff
@@ -80,7 +80,7 @@ class Connection {
      * @return the result of the query.
      */
     template<typename... Args>
-    KUZU_API inline std::unique_ptr<QueryResult> execute(
+    inline std::unique_ptr<QueryResult> execute(
```

Comment:
Should we remove `KUZU_API` from this function? As I remember, this is part of our cpp API.

Reply:
MSVC doesn't let you combine inline functions with `dllimport`/`dllexport` presumably because it doesn't actually do anything (they are defined inline in the header files, not in the library, so there's nothing to export/import). clang and gcc don't seem to care.